lgc/patch/PatchEntryPointMutate.cpp

```diff
@@ -261,20 +269,27 @@ bool PatchEntryPointMutate::runOnModule(Module &module) {
   // Gather user data usage.
   gatherUserDataUsage(&module);
 
+  // Create ShaderInputs object and gather shader input usage.
+  std::unique_ptr<ShaderInputs> shaderInputs = std::make_unique<ShaderInputs>(&module.getContext());
```

Comment:
This should just be a stack-allocated local variable, it doesn't have to be heap-allocated.